app/static/js/courseManagement.js

```diff
@@ -3,13 +3,25 @@ function changeTerm() {
     $('form').submit();
 };
 
+function courseAction(action){
+  courseID = action.id;
```

Comment:
This is a global variable. You should use `var` or `let` to scope it to the function